po/es_NI.po

```diff
@@ -923,33 +929,33 @@ msgstr "URL opciones-gnc: mal formados."
 #: ../gnucash/gnome/top-level.c:98
 #, c-format
 msgid "Bad URL: %s"
-msgstr ""
+msgstr "URL errónea: %s"
 
 #: ../gnucash/gnome/business-urls.c:82
 #, c-format
 msgid "No such entity: %s"
-msgstr ""
+msgstr "No existe la entidad"
 
 #. =================================================================
 #: ../gnucash/gnome/business-urls.c:170
 #, c-format
 msgid "No such owner entity: %s"
-msgstr ""
+msgstr "No existe el dueño de la entidad"
```

Comment:
"dueño de la entidad" would be "entity owner", but that's different from "owner entity", right?

How about "entidad de dueño"?